dash.texi

```diff
@@ -3582,6 +3585,36 @@ plist-like key-value pairs, similarly to &keys keyword of
 This binds @var{n} values from the list to a1 ... aN, then interprets
 the cdr as a plist (see key/value matching above).
 
+@var{a} shorthand notation for kv-destructuring exists which allows the
+patterns be optionally left out and derived from the key name in
```

Comment:
"patterns to be"  :)

Reply:
I'm not an expert on the English subjunctive, but I think a problem here is that you use "exists" followed by "allows," which at least implies the indicative mood.  e.g. this from the Wikipedia article (I hate to cite Wikipedia, but a few minutes of googling didn't find a better source yet):

> The present subjunctive is identical to the bare infinitive (and imperative) of the verb in all forms. This means that, for almost all verbs, the present subjunctive differs from the present indicative only in the third person singular form, which lacks the ending -s in the subjunctive.

I'm sure that one could find archaic "English" uses of nearly anything imaginable (who makes the rules, anyway?), but as a native speaker, "which allows the patterns be" feels ungrammatical at worst, and very awkward at best.  :)

Reply:
Also, "optionally" seems redundant.

Reply:
The subjunctive clause is `allows be` not `exists allows`, but I agree and have no problem to change it.

Reply:
Also for some reason I had the impression you were german... I guess it's the github picture, it looks  very similar to one german user I know :D Oh the silly human brain.

Reply:
Haha, no, I'm American.